src/OrchardCore.Themes/TheAdmin/Assets/scss/TheAdmin.scss

```diff
@@ -1139,28 +1139,46 @@ ul.pager li a {
 .related {
     @extend .float-right;
 }
-// Responsive sizes
-// Creates mixins to target all breakpoints
```

Comment:
These mixins have been removed, as they are now in bootstrap, and the bootstrap versions are better. These where overriding the bootstrap ones, and making auto column arranging fail, i.e. 
```
<div class="col"></div>
<div class="col"></div>